changelog_generator/tag_manager.py

```diff
@@ -78,7 +78,7 @@ class PrefixedTagManager(BaseTagManager):
     """
 
     PATTERN = re.compile(
-        r"^(?P<prefix>\w+)/(?P<major>\d+)\.(?P<minor>\d+)\.((?P<bug>\d+)|rc(?P<rc>\d+))?$"
+        r"^(?P<prefix>[a-zA-Z-_]+)/(?P<major>\d+)\.(?P<minor>\d+)\.((?P<bug>\d+)|rc(?P<rc>\d+))?$"
```

Comment:
Note that this removes digits, since « Matches Unicode word characters; this includes most characters that can be part of a word in any language, as well as numbers and the underscore. »

Does 
```suggestion
        r"^(?P<prefix>[\w-]+)/(?P<major>\d+)\.(?P<minor>\d+)\.((?P<bug>\d+)|rc(?P<rc>\d+))?$"
```
work as expected ? (i.e. word characters + dash)